profiles/car.lua

```diff
@@ -84,7 +84,8 @@ local profile = {
   restricted_access_tag_list = Set {
     'private',
     'delivery',
-    'destination'
+    'destination',
+    'customers',
```

Comment:
Related: https://github.com/Project-OSRM/osrm-backend/issues/3745

We should probably just add the fee tag there, too.